pkg/apis/core/types_shoot.go

```diff
@@ -706,6 +706,8 @@ type Volume struct {
 	VolumeSize string
 	// Encrypted determines if the volume should be encrypted.
 	Encrypted *bool
+	// Interface is the interface of local ssd disk.
+	Interface *string
```

Comment:
Should we call this `LocalSSDInterface` to make it more clear what this field is used for?